src/ORM/Connect/PDOConnector.php

```diff
@@ -338,7 +339,7 @@ public function bindParameters(PDOStatement $statement, $parameters)
             // Check type of parameter
             $type = $this->getPDOParamType($phpType);
             if ($type === PDO::PARAM_STR) {
-                $value = strval($value);
+                $value = (string) $value;
```

Comment:
Do we normally do `(string) $value` or `(string)$value`? i.e. should the space be there?

Reply:
...the space, as is here, seems more common.